### PR TITLE
Simple sv alt allele

### PR DESCRIPTION
**Pull request by SHuang-Broad**
### Description

__Simple__ alt allele constants for SV.
Two implementations, not sure which one is preferred.

### Checklist

- [ ] Code compiles correctly
- [ ] New tests covering changes and new functionality
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary
- [ ] Is not backward compatible (breaks binary or source compatibility)

### CODE REVIEW

**Comment by SHuang-Broad** [comment]
@droazen please review. 
@LeeTL1220 please see this covers what you need.

**Comment by LeeTL1220** [comment]
@SHuang-Broad Yes, I just needed the constants.

File: src/main/java/htsjdk/variant/variantcontext/Allele.java [hunk 1]
```diff
@@ -200,6 +200,11 @@ protected Allele(final Allele allele, final boolean ignoreRefState) {
     public final static Allele NO_CALL = new Allele(NO_CALL_STRING, false);
     public final static Allele NON_REF_ALLELE = new Allele(NON_REF_STRING, false);
 
+    public final static Allele SV_SIMPLE_DEL = new Allele(createBracketedSymbAlleleString(StructuralVariantType.DEL), false);
```
Comment (droazen) [inline src/main/java/htsjdk/variant/variantcontext/Allele.java]: Can you just call `toSymbolicAltAllele()` on the enum values directly, and get rid of `createBracketedSymbAlleleString()`?

File: src/main/java/htsjdk/variant/variantcontext/Allele.java [hunk 2]
```diff
@@ -200,6 +200,11 @@ protected Allele(final Allele allele, final boolean ignoreRefState) {
...
+    public final static Allele SV_SIMPLE_INS = new Allele(createBracketedSymbAlleleString(StructuralVariantType.INS), false);
+    public final static Allele SV_SIMPLE_INV = new Allele(createBracketedSymbAlleleString(StructuralVariantType.INV), false);
+    public final static Allele SV_SIMPLE_CNV = new Allele(createBracketedSymbAlleleString(StructuralVariantType.CNV), false);
+    public final static Allele SV_SIMPLE_DUP = new Allele(createBracketedSymbAlleleString(StructuralVariantType.DUP), false);
```
Comment (droazen) [inline src/main/java/htsjdk/variant/variantcontext/Allele.java]: Can you add a comment above each SV allele declaration, describing what kind of variant it represents?

**Comment by SHuang-Broad** [comment]
@droazen  please review again.

File: src/main/java/htsjdk/variant/variantcontext/Allele.java [hunk 3]
```diff
@@ -200,6 +199,16 @@ protected Allele(final Allele allele, final boolean ignoreRefState) {
     public final static Allele NO_CALL = new Allele(NO_CALL_STRING, false);
     public final static Allele NON_REF_ALLELE = new Allele(NON_REF_STRING, false);
 
+    // for simple deletion, e.g. "ALT==<DEL>" (not that the spec allows, for now at least, alt alleles like <DEL:ME>)
```
Comment (SHuang-Broad) [inline src/main/java/htsjdk/variant/variantcontext/Allele.java]: for myself: "not" -> "note"

**Comment by droazen** [comment]
:+1: looks good to me -- @lbergelson can you merge?

**Pull request merged**